src/config/styles.json

```diff
@@ -1,4 +1,10 @@
 [
+  {
```

Comment:
Can this be sorted alphabetically to avoid favoritism?

Reply:
Done.